rudder-core/src/main/scala/com/normation/rudder/hooks/RunHooks.scala

```diff
@@ -210,10 +212,18 @@ object RunHooks {
            file match {
              case f if (f.isDirectory) => None
              case f =>
-               if (f.canExecute) {
-                 Some(f.getName)
+               if(f.canExecute) {
+                 val name = f.getName
+                 //compare ignore case (that's why it's a regienMatches) extension and name
+                 ignoreSuffixes.find(suffix => name.regionMatches(true, name.length - suffix.length, suffix, 0, suffix.length)) match {
+                   case Some(suffix) =>
+                     HooksLogger.debug(s"Ignoring hook '${f.getAbsolutePath}' because extension '${suffix}' is in the ignore list")
```

Comment:
I would prefer we used "suffix" consistently, rather than "extension". But this is a debug message so maybe not crucial to change, it's just a risk of confusion.